docs/content/en/docs/getting-started/production-environment/_index.md

```diff
@@ -75,33 +75,60 @@ Make sure you use single quotes around the values so that your shell does not in
+
+Post v0.9.0 release of eksctl anywhere, there is an **optional** support for including **curated packages** to the cluster installation.
+
+#### Cluster creation  without curated packages installation
+
+Create the initial cluster
+
+   ```bash
+   # Create a cluster without curated packages installation
+   eksctl anywhere create cluster -f eksa-mgmt-cluster.yaml
+   ```
+
+#### Cluster creation with optional curated packages
+{{% alert title="Note" color="primary" %}}
+* It is *optional* to install the curated packages as part of the cluster creation.
+* `eksctl anywhere version` version should be `v0.9.0` or later.
+* Post-creation installation and detailed package configurations can be found [here.]({{< relref "../../tasks/packages" >}})
+* The EKS Anywhere package controller and the EKS Anywhere Curated Packages (referred to as “features”)
+  are provided as “preview features” subject to the AWS Service Terms,
+  (including Section 2 (Betas and Previews)) of the same. During the EKS Anywhere Curated Packages Public Preview, 
+  the AWS Service Terms are extended to provide customers access to these features free of charge.
+  These features will be subject to a service charge and fee structure at ”General Availability“ of the features.
+  {{% /alert %}}
+1. Discover curated-packages to install
+   ```bash
+   eksctl anywhere list packages --source registry --kube-version 1.21
+   ```
+   Example command output
+   ```                 
+   Package                 Version(s)                                       
+   -------                 ----------                                       
+   harbor                  2.5.0-4324383d8c5383bded5f7378efb98b4d50af827b
+   ```
+2. Generate a curated-packages config
+
    The example shows how to install the `harbor` package from the [curated package list]({{< relref "../../reference/packagespec" >}}).
    ```bash
-   eksctl anywhere generate package harbor --source registry --kubeversion 1.21 > packages.yaml
+   eksctl anywhere generate package harbor --source registry --kube-version 1.21 > packages.yaml
    ```
 
-1. Create the initial cluster
+3. Create the initial cluster
 
-   After you have created your `eksa-mgmt-cluster.yaml` and set your credential environment variables, you will be ready to create the cluster:
    ```bash
-   # Create a cluster without curated packages installation
-   eksctl anywhere create cluster -f eksa-mgmt-cluster.yaml
    # Create a cluster with curated packages installation
    eksctl anywhere create cluster -f eksa-mgmt-cluster.yaml --install-packages packages.yaml
    ```
 
-1. Once the cluster is created you can use it with the generated `KUBECONFIG` file in your local directory:
+### Once the cluster is created you can use it with the generated `KUBECONFIG` file in your local directory:
 
    ```bash
    export KUBECONFIG=${PWD}/${CLUSTER_NAME}/${CLUSTER_NAME}-eks-a-cluster.kubeconfig
    ```
-1. Check the cluster nodes:
+### Check the cluster nodes:
```

Comment:
Can you do this changing number list to headings in another PR. Product made it this way for whatever reason and this is a bigger convo

Reply:
Ok I just made this change